tutorials/requirements.txt

```diff
@@ -0,0 +1,3 @@
+torch
```

Comment:
all of them are optional.
And torch installation will be checked before running any code from pre/post processing.